docs/services.md

```diff
@@ -377,7 +377,7 @@ SDK returned.
   - `config.position` **(`"auto"` \| [Object][150])** If `"auto"`, the viewport will fit the
       bounds of the overlay(s). Otherwise, the maps' position is described by an object
       with the following properties:
-      `coordinates` (required): [`coordinates`][144] for the center of image.
+      `coordinates` (required): [`coordinates`][106] for the center of image.
```

Comment:
Why is this changing?

Reply:
good catch! This PR got old enough that I needed to catch up to master and update the docs.